src/views/award.ts

```diff
@@ -14,54 +14,68 @@
  * limitations under the License.
  */
 
-import { AwardNode } from '@manuscripts/transform'
+import { ContextMenu, ContextMenuProps } from '@manuscripts/style-guide'
+import { AwardNode, schema } from '@manuscripts/transform'
 
-import { Trackable } from '../types'
+import { AwardModal, AwardModalProps } from '../components/awards/AwardModal'
+import {
+  DeleteAwardDialog,
+  DeleteAwardDialogProps,
+} from '../components/awards/DeleteAwardDiaolog'
+import { isDeleted } from '../lib/track-changes-utils'
+import { updateNodeAttrs } from '../lib/view'
+import { Trackable, TrackableAttributes } from '../types'
 import BlockView from './block_view'
 import { createNodeView } from './creators'
+import ReactSubView from './ReactSubView'
 
+export type AwardAttrs = TrackableAttributes<AwardNode>
```

Comment:
I think we should bring back `initialise`, without the `contentDOM`:

```
public initialise() {
    this.createDOM()
    this.updateContents()
  }
```

We do not really need a `contentDOM` in this case and we can replace all usages here with `dom`.

Reply:
it is done this way because of styling and tracking changes styling... parent 'awards' was update to extend BaseNodeView...

Reply:
The TC styling should work without using contentDOM. what do you see if you use dom directly?